src/com/yetanalytics/lrs_admin_ui/functions.cljs

```diff
@@ -1,8 +1,5 @@
 (ns com.yetanalytics.lrs-admin-ui.functions
-  (:require [re-frame.core                    :refer [subscribe dispatch-sync]]
-            [com.yetanalytics.lrs-admin-ui.db :as db]
-            [clojure.pprint :refer [pprint]]))
-
+  (:require [re-frame.core :refer [subscribe dispatch-sync]]))
 
```

Comment:
The `re-frame.core` functions aren't used in this namespace either.

Reply:
fixed